ci-operator/step-registry/hypershift-qe/aws/hypershift-qe-aws-chain.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "hypershift-qe/aws/hypershift-qe-aws-chain.yaml",
```

Comment:
I think the hypershift can be shared with different teams, not only for qe team, right? if right, pls rename `hypershift-qe` to `hypershift`

Reply:
there is already a hypershift dir created by dev team.  I think @LiangquanLi930 uses this dir for some hypershift qe scripts e.g. ginkgo hypershift workflows/preconditions that are not available for dev's dir.

Reply:
`ginkgo hypershift workflows/preconditions` means e2e testing, right? From the content of hypershift-qe-aws-commands.sh, no much is specific only for QE. If possible, QE and dev should contribute codes for one shared place.